### PR TITLE
Added quote on Avril Lavigne

### PR DESCRIPTION
**Pull request by Misha-Mad**
## ✅ New Quote Checklist
- [x] The quote(s) added are in alphabetical order and not ❌ duplicates.
- [x] The quote(s) added are about **learning a foreign language**.
- [x] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [x] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -47,6 +47,10 @@
     "author": "Armenian Proverb",
     "sentence": "You are as many a person as languages you know."
   },
+  {
+    "author": "Avril Lavigne",
+    "sentence": "As humans we speak one language..."
```
Comment (Kvaibhav01) [inline server/data/quotes.json:52]: Please end the sentence with a period.
```suggestion
    "sentence": "As humans we speak one language."
```

**Review by Kvaibhav01 — COMMENTED** [review]
Thanks for the PR @Misha-Mad, update the changes as shown.

File: server/data/quotes.json (hunk 1 above)
Comment (Misha-Mad) [reply server/data/quotes.json]: thanks @Kvaibhav01

**Review by jayehernandez — APPROVED** [review]

**Comment by jayehernandez** [comment]
Thank you for contributing to Letra ☀️, @Misha-Mad!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Pull request merged**